support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -294,35 +288,40 @@ EventsAndTriggers:
       - ErrorIfDataTooBig:
           Threshold: 100
           VariablesToCheck: [SpacetimeMetric]
+  # Observe apparent horizons often enough so they find the next horizon based
+  # on the previous initial guess.
```

Comment:
Please add that this is an issue with the AH finders not sharing initial guesses and that it needs to be fixed.